src/content/protocols/compound-v2.md

```diff
@@ -0,0 +1,338 @@
+| CErc20Delegate                              | sweepToken                      | Moves tokens accidentally sent to the CToken contract to its admin, i.e., the Timelock. A rogue admin could repeatedly call sweepToken to empty out significant balances of arbitrary ERC-20 tokens held by the contract.                                                                                                                                                                          | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate                              | \_delegateCompLikeTo            | Lets admin change the address that receives COMP-style reward distributions. A rogue admin could redirect all COMP rewards to an attacker‐controlled address.                                                                                                                                                                                                                                      | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate / CErc20 / CEther            | \_setPendingAdmin               | Writes a new pendingAdmin address. Controls who can then call `_acceptAdmin` to become the new admin.                                                                                                                                                                                                                                                                                              | Timelock (DAO)                      |
```

Comment:
Updates the admin of the contract. The new admin can [... capabilities]. The new admin has to call `_acceptAdmin` to accept the nomination.